_posts/2018-7-18-EIA-coal.md

```diff
@@ -0,0 +1,141 @@
+</div>
+{% endraw %}
+<sup>_Coal Mines, Powder River Basin, Wyoming, [Google Earth Engine Timelapse](https://earthengine.google.com/timelapse/#v=43.60679,-105.24646,9.613,latLng&t=2.96){:target="_blank"}_</sup>
+
+## Sulfur Content
+
+The EIA data on coal shipments has incredible resolution, and one can find information about quarterly shipments between individual mines and power plants.  For each of these there is information about the type of coal, ash content, heat content, price, quantity, and sulfur content.
+
+The sulfur content of coal is a concern due to [SO<sub>2</sub> pollution](https://www.epa.gov/so2-pollution/sulfur-dioxide-basics#effects){:target="_blank"} resulting from coal combustion, which can lead to problems such as acid rain, respiratory problems, and atmospheric haze.  While high sulfur content does not necessarily translate into high SO<sub>2</sub> emissions due to [desulfurization technology](https://en.wikipedia.org/wiki/Flue-gas_desulfurization){:target="_blank"} used by power plants to reduce emissions[^eGRID_Sulfur], the process of desulfurization is an economic cost rather than a benefit, and examining sulfur content can at a minimum give indications related to the [economics of coal prices](https://www.eia.gov/energyexplained/index.php?page=coal_prices){:target="_blank"}.
+
+The plot below gives an overview of how the coal produced each year differs in the amount of sulfur content.  To construct the plot, we did the following:
+
+* Per year, sort all coal shipments from those with the highest percentage of sulfur to those with the lowest.
+* Using this order, calculate the cumulative quantity.  The last value is the total amount of coal produced in the US in that year.
+
+A useful feature of this type of plot is that the area under the curve is the total amount of sulfur contained in coal shipments that year.  Instead of reducing the yearly amount of sulfur to a single number, this plot shows how it is distributed based on the properties of the coal shipped.
+
+![Coal Sulfur]({{ site.baseurl }}public/images/YearlyCoalSulfurQuantity.gif)
+<sup>_Profile of coal sulfur content._</sup>
+
+For reference, we use 2008 (in blue) as a baseline since that is the first year in the EIA data.  As the animation progresses, we can see that total coal production peaks in 2010, before steadily decreasing to levels below 2008.  By examining the differences between the two curves, we can see where increases and decreases sulfur from different types of coal have occurred.
+
+For example, on the left side of the plot, the gray areas show increased amount of sulfur from coal which is high in sulfur.  Later in the animation, we see light blue areas, representing decreased amounts of sulfur from low-sulfur coal (and less coal production overall).  By subtracting the size of the light blue areas from the gray areas, we can calculate the overall change in sulfur, relative to 2008.
```

Comment:
"For example, on the left side of the plot, the gray areas show increased amount of sulfur from coal which is high in sulfur."
I find this sentence a bit confusing, I think partly because of the double mention of sulful.  I don't have any suggestions for this atm; any ideas?